src/ingestion/destinations/monosi.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, Session
+from typing import Any, List
+
+from ingestion.sources.postgresql import PostgreSQLSourceConfiguration
+
+from .base import Destination, Publisher
+
+
+class SQLAlchemyPublisher(Publisher):
+    def __init__(self, configuration):
+        self.configuration = configuration
+        self.engine = None
+        self.connection = None
+
+    def _create_engine(self):
+        try:
+            return create_engine(self.configuration.connection_string())
+        except Exception as e:
+            raise e
```

Comment:
If we are expecting this to bubble up to a higher level try-catch, we should just use `raise`